packages/docusaurus-theme-translations/locales/tk/theme-common.json

```diff
@@ -0,0 +1,74 @@
+{
+  "theme.AnnouncementBar.closeButtonAriaLabel": "Ýapmak",
+  "theme.BackToTopButton.buttonAriaLabel": "Başyna süýşmek",
+  "theme.CodeBlock.copied": "Göçürildi",
+  "theme.CodeBlock.copy": "Göçürmek",
+  "theme.CodeBlock.copyButtonAriaLabel": "Bufere göçürmek",
+  "theme.CodeBlock.wordWrapToggle": "Setir boýy geçir",
+  "theme.DocSidebarItem.collapseCategoryAriaLabel": "Gapdal paneli gizle '{label}'",
+  "theme.DocSidebarItem.expandCategoryAriaLabel": "Gapdal paneli görkez'{label}'",
+  "theme.ErrorPageContent.title": "Sahypada näsazlyk ýüze çykdy.",
+  "theme.ErrorPageContent.tryAgain": "Gaýtadan synanyşyň",
+  "theme.NavBar.navAriaLabel": "Esasy",
+  "theme.NotFound.p1": "Gynansakda, ýüzlenen sahypaňyz tapylmady",
+  "theme.NotFound.p2": "Web sahypanyň dolandyryjylaryna habar bermegiňizi sizden haýyş edýäris",
+  "theme.NotFound.title": "Sahypa tapylmady",
+  "theme.TOCCollapsible.toggleButtonLabel": "Bu sahypanyň mazmuny",
+  "theme.admonition.caution": "duýduryş",
+  "theme.admonition.danger": "howp",
+  "theme.admonition.info": "maglumat",
+  "theme.admonition.note": "bellik",
+  "theme.admonition.tip": "maslahat",
+  "theme.admonition.warning": "duýduryş",
+  "theme.blog.archive.description": "Arhiw",
+  "theme.blog.archive.title": "Arhiw",
+  "theme.blog.paginator.navAriaLabel": "Blog sahypasynda nawigasiýa",
+  "theme.blog.paginator.newerEntries": "Täze ýazgylar",
+  "theme.blog.paginator.olderEntries": "Köne ýazgylar",
+  "theme.blog.post.paginator.navAriaLabel": "Blog ýazgysynyň sahypasyndaky nawigasiýa",
+  "theme.blog.post.paginator.newerPost": "Täze ýazgy",
+  "theme.blog.post.paginator.olderPost": "Köne ýazgy",
+  "theme.blog.post.plurals": "{count} ýazgy|{count} ýazgylar|{count} ýazgylary",
+  "theme.blog.post.readMore": "Dowamyny okamak",
+  "theme.blog.post.readMoreLabel": "{title} hakda doly okamak",
+  "theme.blog.post.readingTime.plurals": "{readingTime} min okamak|{readingTime} min okamak|{readingTime} min okamak",
```

Comment:
2 forms instead of 3?

Reply:
here plural form doesn't change itself, but as grammatically plural changes according to cases I thought it would be correct to put the 3rd form regardless of its change